scripts/run-analysis.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+sh -c "$*"
```

Comment:
This is really simple. It'll just take any arguments defined the current step in [.github/workflows/run-analysis.yml](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/183/files#diff-6f60327802f48f21c83edf1c43fdc1b2bcc69434ab8e14bfd7a5e7bc75ce6e7a) and run them as shell scripts inside the container.